plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -1659,7 +1690,7 @@ public void registerCoreMembers() {
         // Note: Be careful, gamerules are CASE SENSITIVE.
         //
         // @Tags
-        // TODO: Add tags and then document them!
+        // No tags currently exist

```

Comment:
Open issue/PR for adding tags `<w@world.list_gamerules>` and `<w@world.gamerule[<gamerule>]>` using methods from https://hub.spigotmc.org/javadocs/spigot/org/bukkit/World.html

Reply:
A todo that says "add things!" shouldn't be removed and swapped for "We have nothing"... as merg said, open an issue to get that added... or just leave the todo in place.